tools/ports/libjpeg.py

```diff
@@ -16,7 +16,10 @@ def needed(settings):
 
 
 def get(ports, settings, shared):
-  ports.fetch_project('libjpeg', 'https://www.ijg.org/files/jpegsrc.v9c.tar.gz', 'jpeg-9c', sha512hash=HASH)
+  # Archive mirrored from http://www.ijg.org/files/jpegsrc.v9c.tar.gz.
+  # We have issues where python urllib was not able to load from the www.ijg.org webserver
+  # and was resulting in 403: Forbidden.
+  ports.fetch_project('libjpeg', 'http://www.ijg.org/files/jpegsrc.v9c.tar.gz', 'jpeg-9c', sha512hash=HASH)
```

Comment:
it looks like this just changes the https to http - is that intentional?

Reply:
Oops.. i uploaded the wrong change.. try now :)